app/models/mixins/miq_set_member_mixin.rb

```diff
@@ -0,0 +1,19 @@
+module MiqSetMemberMixin
+  extend ActiveSupport::Concern
+  included do
+    has_many :miq_set_memberships, :dependent => :delete_all, :as => :member
```

Comment:
If we go with membership sti here, defining the class name would be better.

going from set to members requires a stop in ruby land since you can't join across polymorphic.
I had thought that we would get contradictory results where multiple sets would come back for the same id but having unique ids seem to be all set here.

Not being able to properly `includes()` these frustrates me.